_layouts/default.html

```diff
@@ -63,6 +63,60 @@
         <script defer src="/js/bootstrap.min.js"></script>
         <script defer src="/js/scripts.js"></script>
 
+        <!-- Schema.org JSON-LD Markup for Search Engine Optimization -->
+        <script type='application/ld+json'>
+        {
+            "@context": "https://schema.org/",
+            "@type": "SoftwareApplication",
+            "applicationCategory": "FinanceApplication",
+            "name": "Bisq",
+            "image": "https://bisq.network/images/bisq-logo.svg",
+            "availableOnDevice": [
+                "Desktop",
+                "Smartphone",
+                "Tablet"
+            ],
+            "author": {
+            "@type": "Organization",
+            "url": "https://bisq.network/dao/",
+            "name": "Bisq Decentralized Autonomous Organization",
+            "sameAs": [
+                "https://github.com/bisq-network",
+                "https://twitter.com/bisq_network",
+                "https://www.youtube.com/c/bisq-network",
+                "https://bisq.network/slack-invite",
+                "https://bisq.community/",
+                "https://lists.bisq.network/listinfo/bisq-contrib",
+                "https://webchat.freenode.net/?channels=bisq",
+                "https://www.reddit.com/r/bisq",
+                "https://keybase.io/team/bisq"
+	          ]
+            },
+            "downloadUrl": "https://bisq.network/downloads/",
+            "fileSize": "180MB",
+            "operatingSystem": [
+                "Windows",
+                "Linux",
+                "OSX",
+                "iOs",
+                "Android"
+            ],
+            "releaseNotes": "https://github.com/bisq-network/bisq/releases",
+            "screenshot": "https://bisq.network/images/bisq_screen0.svg",
+            "softwareVersion": "{{ site.client_version }}",
+            "aggregateRating": {
```

Comment:
Where's this rating data from? If it's a placeholder/dummy data then I think it's better to omit it.

Reply:
`aggregateRating` is a **required property** for `SoftwareApplication`.